### PR TITLE
added the monitoring tool README

### PR DESCRIPTION
**Pull request by yt3liu**
Fixes #

### CODE REVIEW

File: tools/monitoring/mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Monitoring Database
+
+The monitoring database is a MySQL instance on Cloud SQL. It stores the error logs for each prow job
+failures; and the alert status for all error patterns.
+
+## Database Connection With Cloud Shell
+
```
Comment (steuhs) [inline tools/monitoring/mysql/README.md]: Select project first? The instance is only in knative-tests

File: tools/monitoring/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Navigate to SQL in Google Cloud Platform. Click on the knative-monitoring instance. In the Overview tab,
```
Comment (steuhs) [inline tools/monitoring/mysql/README.md]: is this the "monitoring" cluster mentioned in "Create the cluster"? Or are you talking about the existing one in knative-tests?

File: tools/monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### Create the cluster
+
+```bash
+gcloud container clusters create monitoring --enable-ip-alias
+```
+
+### Secrets
+
+* `monitoring-db-credentials`: monitoring database user name and password
```
Comment (steuhs) [inline tools/monitoring/README.md]: Where is this var used?

File: tools/monitoring/mysql/README.md (hunk 2 above)
Comment (yt3liu) [reply tools/monitoring/mysql/README.md]: The instance here is referring to the Cloud SQL Instance which is named 'knative-monitoring'. This is different from the cluster instance.

File: tools/monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Knative Monitoring
```
Comment (srinivashegde86) [inline tools/monitoring/README.md]: doc header

File: tools/monitoring/mysql/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Monitoring Database
```
Comment (srinivashegde86) [inline tools/monitoring/mysql/README.md]: doc header

File: tools/monitoring/README.md (hunk 4 above)
Comment (yt3liu) [reply tools/monitoring/README.md]: Is there a way to add doc headers to the README.md so it doesn't show up when the md file is rendered? 

I couldn't find a README.md file that has the doc header. 
https://raw.githubusercontent.com/knative/test-infra/master/ci/prow/README.md

What would be the right way to add this? License info in plain text before the first heading?

Comment (srinivashegde86) [reply tools/monitoring/README.md]: oops. did not realize that. ignore my comment..

File: tools/monitoring/README.md (hunk 3 above)
Comment (srinivashegde86) [reply tools/monitoring/README.md]: is this the mounted secret?

Comment (yt3liu) [reply tools/monitoring/README.md]: Sorry missed this comment. Yes this is the secret name that you'll see when you run `kubectl get secrets`

File: tools/monitoring/mysql/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+The monitoring database is a MySQL instance on Cloud SQL. It stores the error logs for each prow job
+failures; and the alert status for all error patterns.
+
+## Database Connection With Cloud Shell
+
+Navigate to SQL in Google Cloud Platform. Click on the knative-monitoring instance. In the Overview tab,
+under Connect to this Instance section, click on Connect using Cloud Shell.
```
Comment (steuhs) [inline tools/monitoring/mysql/README.md]: I do not see " Connect using Cloud Shell" under "Connect to this Instance section". Maybe the UI changes?

I would suggest using quotations or _italic_ when quoting a name phrase

File: tools/monitoring/README.md (hunk 3 above)
Comment (steuhs) [reply tools/monitoring/README.md]: What is the purpose of putting the "secrets" section here? Is it to explain what does the field mean in the yaml, or is it to remind the user to set up the secret? Please be more explicit about it

File: tools/monitoring/mysql/README.md (hunk 6 above)
Comment (yt3liu) [reply tools/monitoring/mysql/README.md]: Changed it to italics and added permission required in each section.

File: tools/monitoring/README.md (hunk 3 above)
Comment (yt3liu) [reply tools/monitoring/README.md]: It is to remind the user to set up the secrets. Removed this section per offline conversation.

**Pull request merged**